src/app/introduction/introduction.component.scss

```diff
@@ -0,0 +1,145 @@
+@import "../styles/components/_variables";
+
+::selection {
+    background: $yellow; /* WebKit/Blink Browsers */
+}
+::-moz-selection {
+    background: $yellow; /* Gecko Browsers */
+}
+h3 {
+    text-align: center;
+    font-weight: $font-bold;
+    color: $blue;
+}
+h2 {
+    font-weight: $font-bold;
+}
+
+.mosquito {
+    width: 80px;
+    height: auto;
+}
+
+.lead-title {
+    font-weight: $font-bold;
+    font-size: 2.5em;
+    color: $black;
+}
+
+.lead-sub-heading {
+    color: grey;
+    font-weight: $font-bold;
+    font-size: 1.5em;
+}
+
+@mixin border-radius($radius1, $radius2) {
```

Comment:
[bootstrap](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/mixins/_border-radius.scss) actually provides border-radius mixins

also, i would use [variable arguments](https://stackoverflow.com/a/15309942) instead for flexibility